pkgs/by-name/db/dbgate/package.nix

```
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  undmg,
+  appimageTools,
+}:
+let
+  pname = "dbgate";
+  version = "5.2.8";
+  src =
+    fetchurl
+      {
+        aarch64-linux = {
+          url = "https://github.com/${pname}/${pname}/releases/download/v${version}/${pname}-${version}-linux_arm64.AppImage";
+          hash = "sha256-gxojSSk7prhnd9fy56B9H+Cj6COBLc7xPfV8dTvSO0c=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/${pname}/${pname}/releases/download/v${version}/${pname}-${version}-linux_x86_64.AppImage";
+          hash = "sha256-/Vfd0R+Mzx1CJKkC7dj99pbuuyh8PJtbYlH3wtwVxSM=";
+        };
+        x86_64-darwin = {
+          url = "https://github.com/${pname}/${pname}/releases/download/v${version}/${pname}-${version}-mac_x64.dmg";
```

```suggestion
        aarch64-linux = {
          url = "https://github.com/dbgate/dbgate/releases/download/v${version}/dbgate-${version}-linux_arm64.AppImage";
          hash = "sha256-gxojSSk7prhnd9fy56B9H+Cj6COBLc7xPfV8dTvSO0c=";
        };
        x86_64-linux = {
          url = "https://github.com/dbgate/dbgate/releases/download/v${version}/dbgate-${version}-linux_x86_64.AppImage";
          hash = "sha256-/Vfd0R+Mzx1CJKkC7dj99pbuuyh8PJtbYlH3wtwVxSM=";
        };
        x86_64-darwin = {
          url = "https://github.com/dbgate/dbgate/releases/download/v${version}/dbgate-${version}-mac_x64.dmg";
```
substituting pname is an anti pattern

---

Didn't know that. Applied suggestion